components/dash-core-components/src/fragments/Markdown.react.js

```diff
@@ -131,7 +134,16 @@ export default class DashMarkdown extends Component {
                 <Markdown
                     source={displayText}
                     escapeHtml={!dangerously_allow_html}
+                    plugins={mathjax ? [RemarkMath] : []}
                     renderers={{
+                        math: props => (
+                            <Math tex={props.value} inline={false} />
+                        ),
+
+                        inlineMath: props => (
+                            <Math tex={props.value} inline={true} />
+                        ),
+
```

Comment:
Thanks to @anders-kiaer for the contribution in :trophy: :trophy: :trophy: https://github.com/plotly/dash-core-components/pull/634 :trophy: :trophy: :trophy: